pkg/cover/report.go

```diff
@@ -132,9 +133,10 @@ func (rg *ReportGenerator) prepareFileMap(progs []Prog) (map[string]*file, error
 	if len(allProgPCs) != len(matchedProgPCs) {
 		for mismatch := range allProgPCs {
```

Comment:
This and the following lines do not seem to be necessary.

Reply:
Indeed.